GenderPayGap.WebUI/Views/ManageOrganisations/AllOrganisationReports.cshtml

```diff
@@ -0,0 +1,329 @@
+﻿@using Newtonsoft.Json
+@using GenderPayGap.Core
+@using GenderPayGap.Core.Classes
+@using GenderPayGap.Core.Helpers
+@using GenderPayGap.WebUI.Models.ManageOrganisations
+@using GenderPayGap.WebUI.Models.Shared
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.ManageOrganisations.AllOrganisationReportsViewModel
+
+@{
+    ViewBag.Title = "View all your employers reports - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";

```

Comment:
Is this layout actually doing anything? I don't think we normally need one.

Reply:
You have to reference a layout to use the `@section BeforeMain` for breadcrumbs so I think I have to specify it.